include/orwell/callbacks/ProcessPong.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "orwell/callbacks/InterfaceProcess.hpp"
+
+namespace orwell
+{
+
+namespace callbacks
+{
+
+class ProcessPong : public InterfaceProcess
+{
+public:
+	ProcessPong(
+			std::shared_ptr< com::Sender > ioPublisher,
```

Comment:
tabulation looks weird in this file too...

Reply:
My current configuration puts two tabulations to indent this. I am not sure if it is bad or good.